core/src/state.rs

```diff
@@ -91,8 +85,8 @@ pub struct World {
 
 /// Struct for block's aggregated changes
 pub struct WorldBlock<'world> {
-    /// Iroha config parameters.
-    pub(crate) parameters: CellBlock<'world, Parameters>,
+    /// Iroha on-chain built-in parameters.
```

Comment:
There are also custom params

```suggestion
    /// On-chain configuration
```